core/src/main/resources/hudson/model/AbstractProject/help-concurrentBuild.html

```diff
@@ -1,24 +1,39 @@
-<div>
-    If this option is checked, Jenkins will schedule and execute multiple builds concurrently (provided
-    that you have sufficient executors and incoming build requests.) This is useful on builds and test jobs that
-    take a long time, as each build will only contain a smaller number of changes, and the total turn-around
-    time decreases due to the shorter time a build request spends waiting for the previous build to complete.
-    It is also very useful with parameterized builds, whose individual executions are independent from each other.
-
-    <p>
-    For other kinds of jobs, allowing concurrent executions of multiple builds may be problematic,
-    for example if it assumes a monopoly on a certain resource, like database, or for jobs where you use Jenkins
-    as a cron replacement.
 
-    <p>
-    If you use a custom workspace and enable this option, all your builds will run on the same workspace,
-    thus unless a care is taken by your side, it'll likely to collide with each other.
-    Otherwise, even when they are run on the same node, Jenkins will use different workspaces to keep
-    them isolated.
-
-    <p>
-    When Jenkins creates different workspaces for isolation, Jenkins appends "@<I>num</I>" to the
-    workspace directory name, e.g. "@2".  The separator "@" can be configured by setting the system property
-    "hudson.slaves.WorkspaceList" to the desired separator string on the Jenkins command line. 
-    E.g. "-Dhudson.slaves.WorkspaceList=-" will use a dash as separator.
+<div>
+  When this option is checked, multiple builds of this project may be executed
+  in parallel.
+  <p>
+  By default, only a single build of a project is executed at a time &mdash; any
+  other requests to start building that project will remain in the build queue
+  until the first build is complete.<br>
+  This is a safe default, as projects can often require exclusive access to
+  certain resources, such as a database, or a piece of hardware.
+  <p>
+  But with this option enabled, if there are enough build executors available
+  that can handle this project, then multiple builds of this project will take
+  place in parallel.  If there are not enough available executors at any point,
+  any further build requests will be held in the build queue as normal.
+  <p>
+  Enabling concurrent builds is useful for projects that execute lengthy test
+  suites, as it allows each build to contain a smaller number of changes, while
+  the total turnaround time decreases as subsequent builds do not need to wait
+  for previous test runs to complete.<br>
+  This feature is also useful for parameterised projects, whose individual build
+  executions &mdash; depending on the parameters used &mdash; can be
+  completely independent from one another.
+  <p>
+  Each concurrently executed build occurs in its own build workspace, isolated
+  from any other builds.  By default, Jenkins appends "<tt>@&lt;num&gt;</tt>" to
```

Comment:
\<tt> is actually obsolete. Prefer using \<code> for example (not sure where/if there's an existing CSS in the core for help texts).